python/src/trezorlib/messages.py

```
@@ -3263,6 +3269,140 @@ def __init__(
+        self.nk = nk
+
+
+class DarkfiSpendDetails(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = None
+    FIELDS = {
+        1: protobuf.Field("value", "uint64", repeated=False, required=True),
+        2: protobuf.Field("token_id", "bytes", repeated=False, required=True),
+        3: protobuf.Field("recipient", "bytes", repeated=False, required=True),
+        4: protobuf.Field("spend_hook", "bytes", repeated=False, required=False, default=None),
+        5: protobuf.Field("user_data", "bytes", repeated=False, required=False, default=None),
+    }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)